test/views/behaviour.js

```diff
@@ -0,0 +1,193 @@
+
+/**
+ * Behaviour of the Echidna testbed.
+ *
+ * @author Antonio <antonio@w3.org>
+ */
+
+'use strict';
+
+var JOBS = ['retrieve-resources', 'specberus', 'token-checker', 'third-party-checker', 'publish', 'tr-install', 'update-tr-shortlink'];
+
+var log;
+var inspector;
+var allSpecs = [];
+var allJobs = {};
+var allMessages = {};
+
+function logMessage(message) {
+
+    log = log || $('pre#console');
+    log.append(new Date().toLocaleTimeString() + '  ' + message + '\n');
+
+}
+
+function dumpObject(obj) {
+
+    inspector = inspector || $('pre#inspector');
+
+    if (!obj) {
+        inspector.text('');
+    }
+    else {
+        inspector.text(JSON.stringify(obj, null, 2));
+    }
```

Comment:
Douglas Crockford recommends that when there is more that one line, for reasons that the JS interpreter isn't always nice when you don't.

And no, I don't think it's harder to read. Actually, I think it's way easier to read when the `if` condition is short (aka when condition + execution < 80 chars). I prefer that billion times over scrolling all over the place when reading a file. Here, the function is **14 lines** when it could be half! Come on...